src/main/java/com/solvd/laba/lab5/factory/MyBatisServiceFactory.java

```diff
@@ -0,0 +1,18 @@
+package com.solvd.laba.lab5.factory;
+
+import com.solvd.laba.lab5.interfaces.MyBatisService;
+import com.solvd.laba.lab5.services.MBAthleteService;
+import com.solvd.laba.lab5.services.MBTeamService;
+
+public class MyBatisServiceFactory {
+
+    public <T> MyBatisService<T> createMBService(String entity) {
+        MyBatisService<T> myBatisService = null;
+        if ("Athlete".equalsIgnoreCase(entity)) {
+            myBatisService = (MyBatisService<T>) new MBAthleteService();
+        } else if ("Team".equalsIgnoreCase(entity)) {
+            myBatisService = (MyBatisService<T>) new MBTeamService();
+        }
+        return myBatisService;
```

Comment:
Update the factory where it can return both dao and mybatis versions of service classes